README.md

```diff
@@ -148,17 +148,18 @@ to attempt to force one or more of us to tamper with the software.
 Create a new API key:
 
 ```sh
-$ turnkey gen --name my-test-key
+$ turnkey gen --keys-folder ~/.config/turnkey/keys --key-name my-test-key --organization $ORGANIZATION_ID
```

Comment:
I can also leave `--keys-folder ~/.config/turnkey/keys` for simplicity

Reply:
I think we should omit as many arguments as possible. The simplest setup is:
```
turnkey gen --organization $ORGANIZATION_ID
```

And in subsequent command we can rely on `turnkey` to select the default key.

We can show how one can create a key with a different name, or in a different folder in a separate doc section perhaps?